packages/core/src/Stores/client-store.js

```diff
@@ -2244,7 +2244,7 @@ export default class ClientStore extends BaseStore {
                     is_social_signup_provider = true;
                     // NOTE: Remove this logic once social signup is intergated with OIDC
                     const loggedState = Cookies.get('logged_state');
-                    if (loggedState === 'false') {
+                    if (loggedState === 'false' && !search_params.get('action') === 'signup') {
```

Comment:
`code` is part of the `action=signup` flow, thus logged state is wrongly set to `true` here
<img width="1117" alt="Screenshot 2025-01-22 at 1 56 16 PM" src="https://github.com/user-attachments/assets/8475447b-5595-4f44-a10f-2deb13160ed1" />